docs/user-guide/release-notes/2.14.x.rst

```diff
@@ -10,3 +10,6 @@ New Features
 
 * New parameter for rsync distributor ``rsync_extra_args`` allows user to 
   specify custom arguments for rsync call during publish operations.
+
+* New publish step RemoveOldRepodataStep removes repodata older than 14 days
```

Comment:
Can a sentence be added here with links to the options saying something like: `Configure this behavior with these options` and link to the options.